python/python/lance/dataset.py

```diff
@@ -1763,9 +1833,18 @@ def create_index(
         if shuffle_partition_concurrency is not None:
             kwargs["shuffle_partition_concurrency"] = shuffle_partition_concurrency
 
+        times = []
+        times.append(time.time())
         self._ds.create_index(
             column, index_type, name, replace, storage_options, kwargs
         )
+        times.append(time.time())
+        logging.info("Final create_index time: %ss", times[1] - times[0])
+        # Save disk space
+        if "precomputed_shuffle_buffers_path" in kwargs.keys() and os.path.exists(
+            kwargs["precomputed_shuffle_buffers_path"]
+        ):
+            shutil.rmtree(kwargs["precomputed_shuffle_buffers_path"])
```

Comment:
this might be an surprising behavior to users. I think this should be left to the user to delete instead of us

Reply:
Done. I log info to the user to consider deleting it themselves now (where can the user see the logs anyway?)